src/components/ExerciseRoutine/CalendarDay/CalendarDay.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+import style from './calendarDay.module.scss';
+
+interface Props {
+  day: string;
+  date: number;
+  didExercise: boolean;
+}
+
+const { s_calendarDay, s_day, s_date, s_today, s_didExercise } = style;
+
+const CalendarDay = ({ day, date, didExercise }: Props) => {
+  const isToday = new Date().getDate() === date;
+  return (
+    <li className={classNames(s_calendarDay, didExercise && s_didExercise, isToday && s_today)}>
```

Comment:
```suggestion
    <li className={classNames(s_calendarDay, {
      [s_didExercise]: didExercise,
      [s_today]: isToday
    })}>
```

classNames 문법을 활용한 방법도 있을것 같습니다~

Reply:
라이브러리에서 제공하는 문법과 일반적으로도 해석 가능한 코드중에 어떤것이 더 나이스한 방법인지 모르겠네요 classNames에서 제공하는 문법을 사용했을때 이점이 있는지 알아보고 있다면 적용해보겠습니다!